test/unit/effects.js

```diff
@@ -1,6 +1,7 @@
 const test = require('tape');
```

Comment:
How would you describe the difference between `test/unit/effects.js` vs `test/unit/effectsTest.js`? Would it make sense to combine these tests into one file?

Reply:
Doesn't seem like there's much difference to me, just combined them!